tabular/src/autogluon/tabular/trainer/abstract_trainer.py

```diff
@@ -347,6 +348,7 @@ def stack_new_level_core(self, X, y, models: Union[List[AbstractModel], dict], X
                     'base_model_paths_dict': base_model_paths,
                     'base_model_types_dict': base_model_types,
                     'random_state': level + self.random_state,
+                    'fold_fitting_strategy': fold_fitting_strategy,
```

Comment:
Should not be here, instead part of ag_args_ensemble

Reply:
Done